vendor-bin/eloquent/composer.json

```diff
@@ -5,8 +5,8 @@
         ]
     },
     "require-dev": {
-        "illuminate/database": "~5.3.0",
-        "illuminate/filesystem": "5.0 - 5.3",
+        "illuminate/database": "^5.3",
```

Comment:
Illuminate is not semver, so a `~` is required as you are allowing updates with BC breaks otherwise